test/test_topic.py

```diff
@@ -402,31 +384,31 @@ def test_msg(self):
 
     def test_msg_cb(self):
         result = self.topic_msg_cb.tick()
-        msg = rospy.wait_for_message('/topic_cb', String)
+        msg = rospy.wait_for_message('topic_cb', String)
```

Comment:
So this test is for the Topic Publisher, so it's making sure an actual topic gets published.  The other one was a Monitor (so it subscribed via a callback).  In that one I'm assuming that ros isn't broken and a ros Subscriber callback is getting appropriately called in the Monitor.  Here, the "topic_cb" is a function that I created, and therefore needs to be tested.